lib/eco/pgviewer.py

```diff
@@ -100,16 +100,50 @@ def add_node(self, node, graph):
             label = "<return>"
         if lookup == "<ws>":
             label = "<ws> x%s" % len(label)
+        if label == "\\":
+            label = "<backslash>"
         label = label.replace("\"", "\\\"")
         label = label.replace("\\\\\"", "\\\\\\\"")
         dotnode = pydot.Node("\"%s\"" % id(node), label='"%s"' % label, shape="box")
 
         try:
-            changed = node.get_attr("changed", self.version)
+            if node.get_attr("nested_errors", self.version):
+                dotnode.set('color','firebrick')
         except AttributeError:
-            changed = node.changed
-        if changed:
-            dotnode.set('color','green')
+            pass
+
+        try:
```

Comment:
I don't think they can. If one fails I don't want all the others to fail as well, i.e just because I can't colour mark a node as `changed` doesn't mean it can't be colour-marked as `deleted` or `isolated`.